docs/data-sources/maintenance_windows.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: "wavefront"
+page_title: "Wavefront: Maintenance Windows"
+description: |-
+    Get the information about all Wavefront maintenance windows.
+---
+
+# Data Source: wavefront_maintenance_window_all
+
+Use this data source to get information about all Wavefront maintenance windows.
+
+## Argument Reference
+* `limit` - (Optional) Limit is the maximum number of results to be returned. Defaults to 100.
+* `offset` - (Optional) Offset is the offset from the first result to be returned. Defaults to 0.
+
+## Example Usage
+
+```hcl
+# Get the information about all external links.
+data "wavefront_maintenance_window_all" "example" {
+  limit = 10
+  offset = 0
+}
+```
+
+## Attribute Reference
+
+* `maintenance_windows` - List of all external links in Wavefront. For each external link you will see a list of attributes.
+    * `id` -  The ID of the maintenance window.
+    * `event_name` - The event name of the maintenance window.
+    * `reason` - The reason for the maintenance window.
+    * `title` - The title of the maintenance window.
+    * `customer_id` - The ID of the customer in Wavefront.
+    * `running_state` - The running state of the maintenance window.
+    * `start_time_in_seconds` - The start time in seconds after 1 Jan 1970 GMT.
+    * `end_time_in_seconds` - The end time in seconds after 1 Jan 1970 GMT.
+    * `relevant_customer_tags` - The list of alert tags whose matching alerts will be put into maintenance because
+      of this maintenance window. At least one of `relevant_customer_tags`, `relevant_host_tags`, or `relevant_host_names`
+      is required.
+    * `relevant_host_tags` - The list of source/host tags whose matching sources/hosts will be put into maintenance
```

Comment:
Please apply the comments from the review of maintenance_window.md.

Reply:
Done! Thank you @Margarita-Staneva !